pkg/image/layer.go

```diff
@@ -0,0 +1,102 @@
+package image
+
+import (
+	"github.com/go-gl/gl/v2.1/gl"
+	"github.com/gregjohnson2017/tabula-editor/pkg/gfx"
+	"github.com/gregjohnson2017/tabula-editor/pkg/log"
+	"github.com/gregjohnson2017/tabula-editor/pkg/ui"
+	"github.com/veandco/go-sdl2/sdl"
+)
+
+type Layer struct {
+	area    sdl.Rect
+	origW   int32
+	origH   int32
+	buffer  *gfx.BufferArray
+	texture gfx.Texture
+}
+
+func NewLayer(offset sdl.Point, texture gfx.Texture) *Layer {
```

Comment:
This file is not being used.

Reply:
view.go uses this, there are pointers to Layer's in the view struct - you might be looking at the "view" file that Tyler made to save the old way thing were being done in view so we can easily reference it. It will be deleted shortly once we figure out what we're doing with selection outlines